src/features/dashboard/Dashboard.tsx

```diff
@@ -1,83 +1,91 @@
-            ]
-        },
-        {
-            value: totalActive === 0 ? t('empty') : totalActive.toString(),
-            content: totalActive === 1 ? t('oneActiveIntegration') : t('activeIntegrations'),
-            links: [
-                {name: t('links.integrations'), href: '/integration/list'}
-            ]
-        },
-        {
-            value: totalDispatched === 0 ? t('empty') : totalDispatched.toString(),
-            content: totalDispatched === 1 ? t('oneInstance') : t('instances'),
-            links: [
-                {name: t('links.instances'), href: '/integration/instance/list'}
-            ]
-        },
-        {
-            value: totalErrors === 0 ? t('empty') : totalErrors.toString(),
-            content: totalErrors === 1 ? t('oneError') : t('errors'),
-            links: [
-                {name: t('links.instances'), href: '/integration/instance/list'}
-            ]
-        }
-    ]
+	const cards: ICard[] = [
+		{
+			value:
+				integrations === undefined || integrations.length === 0
+					? t("empty")
+					: integrations.length.toString(),
+			content:
+				integrations !== undefined && integrations.length === 1
+					? t("oneIntegration")
+					: t("integrations"),
+			links: [{ name: t("links.integration"), href: "/integration/new" }],
+		},
+		{
+			value: totalActive === 0 ? t("empty") : totalActive.toString(),
+			content:
+				totalActive === 1 ? t("oneActiveIntegration") : t("activeIntegrations"),
+			links: [{ name: t("links.integrations"), href: "/integration/list" }],
+		},
+		{
+			value: totalDispatched === 0 ? t("empty") : totalDispatched.toString(),
+			content: totalDispatched === 1 ? t("oneInstance") : t("instances"),
+			links: [
+				{ name: t("links.instances"), href: "/integration/instance/list" },
+			],
+		},
+		{
+			value: totalErrors === 0 ? t("empty") : totalErrors.toString(),
+			content: totalErrors === 1 ? t("oneError") : t("errors"),
+			links: [
+				{ name: t("links.instances"), href: "/integration/instance/list" },
+			],
+		},
+	];
 
-    return (
-        <PageTemplate id={'dashboard'} keyPrefix={'pages.dashboard'} customHeading>
-            <Box id={'dashboard-card-container'} display="flex" position="relative" width={1} height={1}>
-                {cards.map((card: ICard, index) => {
-                    return (
-                        <DashboardCard
-                            key={index}
-                            id={`dashboard-card-` + index}
-                            value={card.value}
-                            content={card.content}
-                            links={card.links}
-                            classes={classes}
-                        />)
-                })}
-            </Box>
-        </PageTemplate>
-    );
-}
+	return (
+		<PageTemplate id={"dashboard"} keyPrefix={"pages.dashboard"} customHeading>
+			<HStack align={"center"} justify={"center"} gap={"16"}>
```

Comment:
denne har wrap som default, sånn at de havner under hverandre om vi krymper vinduet, er det meningen? tror vi har noWrap flere andre steder ihverrtfall for å unngå det

Reply:
![Skjermbilde 2024-01-03 kl  19 24 52](https://github.com/FINTLabs/fint-flyt-integration-frontend/assets/51415032/2ba6ba80-6041-4d49-a70a-79c62c442372)

Reply:
my bad, det er lagt til på branchen for configuration, og det var ikke flere steder, så det må sjekkes opp. og det er wrap={false} ikke noWrap 🤦‍♀️

Reply:
Her gjør den ikke det  . men det vel toggle som ikke godtar det ? 
![Skjermbilde 2024-01-03 kl  19 26 27](https://github.com/FINTLabs/fint-flyt-integration-frontend/assets/51415032/463731ea-c119-4a2b-b97e-09375f8baf98)